src/util.rs

```diff
@@ -34,6 +34,16 @@ pub fn document() -> web_sys::Document {
         .expect("Can't find the window's document")
 }
 
+/// Convenience function to access the `web_sys::HtmlDocument`.
+pub fn html_document() -> web_sys::HtmlDocument {
```

Comment:
Could you provide an example? (I'm just curious what forces you to use `HtmlDocument` instead of `Document`).

Reply:
> Could you provide an example?

sure: `html_document().cookie()`

> (I'm just curious what forces you to use `HtmlDocument` instead of `Document`).

the [`cookie` method](https://docs.rs/web-sys/0.3.29/web_sys/struct.HtmlDocument.html#method.cookie) is only available in `HtmlDocument`.